src/Connections/PhpRedisConnection.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+namespace Monospice\LaravelRedisSentinel\Connections;
+
+use Monospice\LaravelRedisSentinel\Exceptions\RedisRetryException;
+use Closure;
+use Illuminate\Redis\Connections\PhpRedisConnection as LaravelPhpRedisConnection;
+use Redis;
+use RedisException;
+
+/**
+ * Executes Redis commands using the PhpRedis client.
+ *
+ * This package extends Laravel's PhpRedisConnection class to wrap all command
+ * methods with a retryOnFailure method.
+ *
+ * @category Package
+ * @package  Monospice\LaravelRedisSentinel
+ * @author   Jeffrey Zant <j.zant@slash2.nl>
+ * @license  See LICENSE file
+ * @link     https://github.com/monospice/laravel-redis-sentinel-drivers
+ */
+class PhpRedisConnection extends LaravelPhpRedisConnection
+{
+    /**
+     * The connection creation callback.
+     *
+     * Laravel 5 does not store the connector by default.
+     *
+     * @var callable
+     */
+    protected $connector;
+
+    /**
+     * The number of times the client attempts to retry a command when it fails
+     * to connect to a Redis instance behind Sentinel.
+     *
+     * @var int
+     */
+    protected $retryLimit = 20;
```

Comment:
Is this an appropriate default (and is it based on a Redis configuration)? For a queue process, 20 retries with 1 second delay each may be fine, but for a web worker this seems excessive. What is your reasoning behind the value 20?

Reply:
And are defaults really required here, since the `PhpRedisConnection` is instantiated through `PhpRedisConnector` which already provides defaults (and so does the constructor of this class)?

Reply:
It's indeed the same configuration as for the `PredisConnection`. I'll reduce the duplication in this! :)